src/Transport.ts

```diff
@@ -87,12 +91,17 @@ export class Transport {
             this._localHandlerRegistrations.forEach(msg => {
                 this._channel.send(msg)
             })
+
+            // When the far end connects, signal which local handlers are unset
```

Comment:
This appears to be redundant with the removal of all remote handlers on disconnect. Since no remote handlers can be registered until the channel is actually connected, I would even say that there is no need to buffer the registration messages in `_localHandlerUnregistrations`